pkgs/tools/security/passff-host/default.nix

```diff
@@ -1,27 +1,41 @@
 { stdenv, fetchFromGitHub, python3, pass }:
 
 stdenv.mkDerivation rec {
-  name = "passff-host-${version}";
+  pname = "passff-host";
   version = "1.2.1";
 
   src = fetchFromGitHub {
     owner = "passff";
-    repo = "passff-host";
+    repo = pname;
     rev = version;
     sha256 = "0ydfwvhgnw5c3ydx2gn5d7ys9g7cxlck57vfddpv6ix890v21451";
   };
 
   buildInputs = [ python3 ];
+  makeFlags = [ "VERSION=${version}" ];
 
   patchPhase = ''
     sed -i 's#COMMAND = "pass"#COMMAND = "${pass}/bin/pass"#' src/passff.py
   '';
 
+  nativeMessagingPaths = [
```

Comment:
May seen rather nitpicky, but can we have the equivalent to this in `installPhase` as bash?

Reply:
Sure, I'm never sure which seems like better convention and mostly just try to go for whatever seems least invasive. As a variable or inline with the for?

Reply:
[previous head](https://github.com/NixOS/nixpkgs/commit/c73ff37cd5601c144138e1de57be3575bac8bb8e) is something that I do frequently, lots of uses like that in nixpkgs. So I'd say that's ideal.

Reply:
Sure, pushed.